src/plugins/extra/mod.rs

```diff
@@ -15,8 +15,10 @@ pub mod tables;
 pub mod beautify_links;
 #[cfg(feature = "linkify")]
 pub mod linkify;
+pub mod smartquotes;
 #[cfg(feature = "syntect")]
 pub mod syntect;
+pub mod typographer;
```

Comment:
indeed, that is yet to be done. in #4 you remarked that it would make more sense to enable both typographer and smartquotes by default, _once_ they are both ready for prime-time. So I'm keeping this as a to-do in this PR (assuming #4 gets merged first)